cli/src/commands/mod.rs

```diff
@@ -163,12 +164,15 @@ pub fn run_command(ui: &mut Ui, command_helper: &CommandHelper) -> Result<(), Co
     match &subcommand {
         Command::Abandon(args) => abandon::cmd_abandon(ui, command_helper, args),
         Command::Absorb(args) => absorb::cmd_absorb(ui, command_helper, args),
-        Command::Backout(args) => backout::cmd_backout(ui, command_helper, args),
+        Command::Backout(args) => {
+            let cmd = renamed_cmd("backout", "revert", false, backout::cmd_backout);
+            cmd(ui, command_helper, args)
+        }
         #[cfg(feature = "bench")]
         Command::Bench(args) => bench::cmd_bench(ui, command_helper, args),
         Command::Bookmark(args) => bookmark::cmd_bookmark(ui, command_helper, args),
         Command::Branch(args) => {
-            let cmd = renamed_cmd("branch", "bookmark", bookmark::cmd_bookmark);
```

Comment:
Perhaps this should be a new function, e.g. `deprecated_with_replacement(...)`, instead of a boolean argument? In theory, you could also add a more private helper function they both call, thought in  this case it might be simpler to just duplicate the code.

Reply:
Yeah, we ca just insert the warning to `cmd_backout()`.